src/links/index.js

```diff
@@ -1,57 +1,44 @@
 export const PARAMETERS = {
-  invite: "https://repl.it/classroom/invite/XvipfNO",
-  exercise1:
-    "https://repl.it/student_embed/assignment/1492285/0339cc5f2176593cbe1e6a6b6cf40cb2"
+  invite: 'https://repl.it/classroom/invite/XvipfNO',
+  exercise1: 'https://repl.it/student_embed/assignment/1492285/0339cc5f2176593cbe1e6a6b6cf40cb2',
 };
 
 export const RETURN = {
-  invite: "https://repl.it/classroom/invite/Xvb4pIt",
-  exercise1:
-    "https://repl.it/student_embed/assignment/1492272/e1e304a8fbe1dad1eefaddbe897bdae6",
-  exercise2:
-    "https://repl.it/student_embed/assignment/1492271/4257dd0bdc509b2cc123249c1ff02526"
+  invite: 'https://repl.it/classroom/invite/Xvb4pIt',
+  exercise1: 'https://repl.it/student_embed/assignment/1492272/e1e304a8fbe1dad1eefaddbe897bdae6',
+  exercise2: 'https://repl.it/student_embed/assignment/1492271/4257dd0bdc509b2cc123249c1ff02526',
 };
 
 export const ARROW_FUNCTIONS = {
-  invite: "https://repl.it/classroom/invite/XvedktC",
-  exercise1:
-    "https://repl.it/student_embed/assignment/1492279/4905683f6a600f95f4b74d3003cec53a"
+  invite: 'https://repl.it/classroom/invite/XvedktC',
+  exercise1: 'https://repl.it/student_embed/assignment/1492279/4905683f6a600f95f4b74d3003cec53a',
 };
 
 export const CALLBACKS = {
-  invite: "https://repl.it/classroom/invite/XvhmgdL",
-  exercise1:
-    "https://repl.it/student_embed/assignment/1492283/5586c54efd8e2124d7a123f30e8b20f8"
+  invite: 'https://repl.it/classroom/invite/4Zjq5QB',
+  exercise1: 'https://repl.it/student_embed/assignment/2411244/c07f8cfc1c44fffff93dada539f1a94e',
 };
 
 export const PURE_FUNCTIONS = {
-  invite: "https://repl.it/classroom/invite/XvdamSz",
-  exercise1:
-    "https://repl.it/student_embed/assignment/1492278/37a21c5b28239a43b266961c089ed44d"
+  invite: 'https://repl.it/classroom/invite/4Zjq5QB',
+  exercise1: 'https://repl.it/student_embed/assignment/2410962/0daba390c33aaf5bffc4477b5232af85',
 };
 
 export const FILTER = {
-  invite: "https://repl.it/classroom/invite/XvJM11a",
-  exercise1:
-    "https://repl.it/student_embed/assignment/1490944/600947e6a9cee06b8ed387e0b7cecd0c",
-  exercise2:
-    "https://repl.it/student_embed/assignment/1490943/7cf916312a71aab62f56a16b78903406"
+  invite: 'https://repl.it/classroom/invite/4Zjq5QB',
+  exercise1: 'https://repl.it/student_embed/assignment/2410967/d5b7a1013743dbc7155c704b67907cd3',
+  exercise2: 'https://repl.it/student_embed/assignment/1490943/7cf916312a71aab62f56a16b78903406',
 };
 
 export const MAP = {
-  invite: "https://repl.it/classroom/invite/XvIJ2r7",
-  exercise1:
-    "https://repl.it/student_embed/assignment/1490942/91e0113f0ac30c223dfa0f58280ece73",
-  exercise2:
-    "https://repl.it/student_embed/assignment/1490941/0cd4dd9dc924589b1814a3d307e654d8"
+  invite: 'https://repl.it/classroom/invite/4Zjq5QB',
+  exercise1: 'https://repl.it/student_embed/assignment/2410966/4267917d63e71d9fc13e3d24fe0bac68',
```

Comment:
getting the same here...could be a permissions thing on my end? I see they're different links but both go to the `name` mapping exercise

Reply:
i think its the same case as above so ill fix that